src/syscheckd/src/db/testtool/action.h

```diff
@@ -362,13 +362,13 @@ struct SyncTxnRowsAction final : public IAction
 struct GetDeletedRowsAction final : public IAction
 {
     void execute(std::unique_ptr<TestContext>& ctx,
-                 const nlohmann::json& /*value*/) override
+                 const nlohmann::json& value) override
     {
+        const auto options = value.contains("options") ? value["options"] : nlohmann::json::object();
         const auto txnOutputFileName{ ctx->outputPath + "/txn_ops.json" };
-
         auto callbackDelete
         {
-            [txnOutputFileName, &ctx](ReturnTypeCallback type, const nlohmann::json & json)
+            [txnOutputFileName, options, &ctx](ReturnTypeCallback type, const nlohmann::json & json)
```

Comment:
Unless there is a reason not to, we could use `&opt` here to avoid an object copy.